h/accounts/views.py

```diff
@@ -301,8 +301,12 @@ def __init__(self, request):
         tos_link = ('<a href="/terms-of-service">' +
                     _('Terms of Service') +
                     '</a>')
+        cg_link = ('<a href="/community-guidelines">' +
+                   _('Community Guidelines') +
+                   '</a>')
         form_footer = _('You are agreeing to be bound by '
-                        'our {tos_link}.').format(tos_link=tos_link)
+                        'our {tos_link} and {cg_link}.').format(tos_link=tos_link,
```

Comment:
Slight difference from the mockup - https://trello.com/c/r5IwlL5T/185-publish-our-community-guidelines-and-link-them-from-our-signup-module that uses the text "... and _our_ Community Guidelines". This generates the text "...and Community Guidelines" . I prefer the wording as implemented here though.